README.md

```diff
@@ -58,34 +58,34 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 
 
-## Installation using production-ready Docker Images
+## Installation using Docker Images
 
  The following prerequisites are required:
 
-*  a Linux shell - Windows users can use WSL of the Git Bash shell instead
+*  a Linux shell - Windows users can use WSL or the Git Bash shell instead
 *  docker
 *  docker-compose
 
 
 
-### Deployment For Newcomers
+### First Example Deployment
 
-This approach is suggested if you are completely new to CoatRack and want to just want to experience what it actually does. Follow these instructions:
+This approach is suggested if you are familiar with the basic CoatRack functionality provided by [coatrack.eu](https://coatrack.eu) and you want to deploy your own CoatRack instance. Go to the directory `docker/docker-compose-deployment` and follow these instructions:
```

Comment:
Maybe mention that this is an experimental or test deployment?

Reply:
As discussed, I just removed this section from the README for now and saved your suggestions for later when all features are working and this section could be re-added.